### PR TITLE
Read pixel spacing from tif files and use when calculating regionprops

### LINKED ISSUE #52: Units for the cell statistics
I'm not sure what units the cell statistics calculated by `skimage.measure.regionprops_tables` are in, nor what units we want them to be in.

We should also look into displaying the units alongside the values, or if this is not possible then add the units to a tooltip for the 'Calculate statistics' button

### PR DESCRIPTION
**Pull request by p-j-smith**
Fixes #52 

- add `PartSeg` as a dependency - it reads several file formats along with some useful metadata

- PartSeg comes with napari readers for various file formats. However, it [scales the image](https://napari.org/stable/api/napari.layers.Image.html#:~:text=\)%20%E2%80%93%20Layer%20metadata.-,scale,-\(tuple%20of) in the viewer based on the pixel spacing. This would be fine, but after projecting the 3d image to 2d it is very difficult to fine to correct plane that the 2d image is in (because the pixel spacing in `z` much larger than the spacing in `xy`). So I've updated the EpiTools reader to be a wrapper around one of the PartSeg readers that removes the scaling from 

- use the pixel spacing when calculating cell statistics with regionprops. Area and perimeter are then converted to be in  $\micro\text{m}^2$ and $\micro\text{m}^2$, respectively

### CODE REVIEW

File: src/napari_epitools/napari.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ contributions:
   commands:
     - id: napari-epitools.get_reader
       python_name: napari_epitools._reader:napari_get_reader
-      title: Open data with napari EpiTools
+      title: EpiTools (".tif", ".tiff")
```
Comment (paddyroddy) [inline src/napari_epitools/napari.yaml:7]: I find this name less clear

File: src/napari_epitools/_reader.py [hunk 2]
```diff
@@ -5,7 +5,12 @@
 implement multiple readers or even other plugin contributions. see:
 https://napari.org/plugins/guides.html?#readers
 """
+
+import pathlib
+
 import numpy as np
+import PartSegCore.analysis.load_functions
+import PartSegCore.napari_plugins.loader
```
Comment (paddyroddy) [inline src/napari_epitools/_reader.py:13]: Do you refuse to use `from` now? haha

**Review by paddyroddy — APPROVED** [review]
🪂

File: src/napari_epitools/napari.yaml (hunk 1 above)
Comment (p-j-smith) [reply src/napari_epitools/napari.yaml:7]: yeah I'll change it back

File: src/napari_epitools/_reader.py [hunk 3]
```diff
@@ -29,7 +34,12 @@ def napari_get_reader(path):
...
 
     # if we know we cannot read the file, we immediately return None.
-    if not path.endswith(".npy"):
+    supported_extensions = [
+        ".tif",
+        ".tiff",
+    ]
+    extension = pathlib.Path(path).suffix.lower()
+    if extension not in supported_extensions:
         return None
```
[the remark this answers is not in the document]
Comment (paddyroddy) [reply src/napari_epitools/_reader.py:43]: I think it still looks the same?

[the remark this answers is not in the document]
Comment (p-j-smith) [reply src/napari_epitools/_reader.py:43]: oh sorry I edited your comment! I'll go ahead and merge then :)

[the remark this answers is not in the document]
Comment (p-j-smith) [reply src/napari_epitools/_reader.py:43]: argh I forgot to commit that change before merging. I'll submit a pr

**Pull request merged**